<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string>QLabel {
    background-color: #2b2b2b;
    border: 1px solid #555555;
    border-radius: 3px;
    padding: 4px 8px;
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>계약일자</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_contract_date">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string>QLabel {
    background-color: #2b2b2b;
    border: 1px solid #555555;
    border-radius: 3px;
    padding: 4px 8px;
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>신청 유형</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_apply_type">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string>QLabel {
    background-color: #2b2b2b;
    border: 1px solid #555555;
    border-radius: 3px;
    padding: 4px 8px;
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>이름</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_name_1">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string>QLabel {
    background-color: #2b2b2b;
    border: 1px solid #555555;
    border-radius: 3px;
    padding: 4px 8px;
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>생년월일</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_birth_date">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string>QLabel {
    background-color: #2b2b2b;
    border: 1px solid #555555;
    border-radius: 3px;
    padding: 4px 8px;
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>성별</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_gender">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_ent_1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_17">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>기관명</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_18">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>대표자</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_ent_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_19">
       <item>
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>법인등록번호</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_21">
       <item>
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>사업자등록번호</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_23">
       <item>
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>개인사업자명</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string>QLabel {
    background-color: #2b2b2b;
    border: 1px solid #555555;
    border-radius: 3px;
    padding: 4px 8px;
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>차종</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_model">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string>QLabel {
    background-color: #2b2b2b;
    border: 1px solid #555555;
    border-radius: 3px;
    padding: 4px 8px;
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>출고예정일</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_deliver_date">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string>QLabel {
    background-color: #2b2b2b;
    border: 1px solid #555555;
    border-radius: 3px;
    padding: 4px 8px;
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>주소</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_address_1">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_address_2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string>QLabel {
    background-color: #2b2b2b;
    border: 1px solid #555555;
    border-radius: 3px;
    padding: 4px 8px;
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>전화</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_hp">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string>QLabel {
    background-color: #2b2b2b;
    border: 1px solid #555555;
    border-radius: 3px;
    padding: 4px 8px;
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>휴대폰</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_pn">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="styleSheet">
          <string>QLabel {
    background-color: #2b2b2b;
    border: 1px solid #555555;
    border-radius: 3px;
    padding: 4px 8px;
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>이메일</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_email">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="styleSheet">
          <string>QLabel {
    background-color: #2b2b2b;
    border: 1px solid #555555;
    border-radius: 3px;
    padding: 4px 8px;
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>공동명의</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_name_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_birth_date_2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="styleSheet">
          <string>QLabel {
    background-color: #2b2b2b;
    border: 1px solid #555555;
    border-radius: 3px;
    padding: 4px 8px;
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>다자녀</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_children">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="styleSheet">
          <string>QLabel {
    background-color: #2b2b2b;
    border: 1px solid #555555;
    border-radius: 3px;
    padding: 4px 8px;
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>청년생애</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_firstandyouth">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_16">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="styleSheet">
          <string>QLabel {
    background-color: #2b2b2b;
    border: 1px solid #555555;
    border-radius: 3px;
    padding: 4px 8px;
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>기타</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_etc">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_bottom">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_20">
       <item>
        <widget class="QLabel" name="label_26">
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: #2b2b2b;
    border: 1px solid #555555;
    border-radius: 3px;
    padding: 4px 8px;
    font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>RN 번호</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_RN">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_reverse_tool">
     <item>
      <widget class="QLineEdit" name="lineEdit_reverse_tool">
       <property name="placeholderText">
        <string>역순 도구 (스페이스바를 누르면 텍스트가 역순으로 변환됩니다)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
